src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![deny(clippy::all, clippy::pedantic, clippy::cargo, unsafe_code)]
 // #![deny(clippy::missing_docs)]
+#![allow(clippy::multiple_crate_versions)]
```

Comment:
I believe this is fixed in upstream `zbus` as I no longer see these warnings where I did february 10th.

Reply:
zbus v3.10.0 seems to not have solved this problem for me. I'm on latest stable and ran `cargo update` before checking with `cargo clippy`.